### PR TITLE
Audio

### PR DESCRIPTION
**Pull request by Hochart-dev**
add component audio + style

### CODE REVIEW

**Review by CaroD59 — DISMISSED** [review]
Ok ! :)

File: src/components/Play/Play.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        className="iconePlay"
+        type="image"
+        id="play"
+        alt="play"
+        src="/images/icones/play-button.png"
+        onClick={playPause}
+      />
+      <audio className="song" src="/mp3/Stay.mp3" type="audio">
+        <track default kind="captions" />
+        {}
```
Comment (LoicBrassart) [inline src/components/Play/Play.jsx]: Why these empty brackets ?

**Review by LoicBrassart — DISMISSED** [review]
I don't really wannt to refuse this PR for a mere `{}`...
If I let it go, can you **vow** to get rid of it quickly ?

**Review by CaroD59 — APPROVED** [review]
Ok for me !

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**